src/account/AccountFactory.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import {IEntryPoint} from "@eth-infinitism/account-abstraction/interfaces/IEntryPoint.sol";
+
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {Create2} from "@openzeppelin/contracts/utils/Create2.sol";
+
+import {UpgradeableModularAccount} from "../../src/account/UpgradeableModularAccount.sol";
+import {ValidationConfigLib} from "../../src/helpers/ValidationConfigLib.sol";
+import {SingleSignerValidation} from "../../src/plugins/validation/SingleSignerValidation.sol";
+
+contract AccountFactory is Ownable {
+    UpgradeableModularAccount public accountImplementation;
+    bytes32 private immutable _PROXY_BYTECODE_HASH;
+    uint32 public constant UNSTAKE_DELAY = 1 weeks;
+    IEntryPoint public entryPoint;
+    address public self;
+
+    constructor(IEntryPoint _entryPoint, UpgradeableModularAccount _accountImpl) Ownable(msg.sender) {
```

Comment:
Maybe a more general comment, probs missing something, but why are we using OZ's ERC1967 instead of using clones like we do with LightAccount?

Reply:
since this shouldn't be used in prod, just testing, i optimized for dev speed

Reply:
Makes sense to me!